consul/consul.go

```diff
@@ -125,6 +131,35 @@ func (r *ConsulAdapter) buildCheck(service *bridge.Service) *consulapi.AgentServ
 	return check
 }
 
+func (r *ConsulAdapter) PostAttributes(service *bridge.Service) error {
+	for k, v := range service.Attrs {
+		data := new(consulapi.KVPair)
+		data.Key = buildServiceKey(service, k)
+		data.Value = []byte(v)
+		log.Println("Updating Consul KV: ", data.Key, v)
+		_, err := r.client.KV().Put(data, nil)
```

Comment:
registrator already has a consulkv adapter, why not just reuse the code, or exact the common code: https://github.com/gliderlabs/registrator/blob/master/consulkv/consulkv.go